### ISSUE TITLE
Speaker Name Hover on Individual Sessions is Missing Theme

### ISSUE THREAD

**Issue by dzoladz** [code4lib/2021.code4lib.org#143]
Hovering over the speaker name on the individual session screen darkens, when it should be a 10% lighten.
<img width="694" alt="Screen Shot 2021-02-09 at 7 00 03 AM" src="https://user-images.githubusercontent.com/10561752/107361055-c5c2f080-6aa4-11eb-8972-492ca5c8e610.png">
